Public/New-HTMLStatusItem.ps1

```diff
@@ -1,43 +1,72 @@
 function New-HTMLStatusItem {
-    [CmdletBinding()]
+    [CmdletBinding(DefaultParameterSetName = 'Percent')]
     param(
         [string] $ServiceName,
+
         [string] $ServiceStatus,
-        [ValidateSet('Dead', 'Bad', 'Good')]$Icon = 'Good',
-        [ValidateSet('0%', '10%', '30%', '70%', '100%')][string] $Percentage = '100%'
+
+        [ValidateSet('Dead', 'Bad', 'Good')]
+        [string]$Icon = 'Good',
+
+        [ValidateSet('0%', '10%', '30%', '70%', '100%')]
+        [Parameter(ParameterSetName = 'Percent')]
+        [string]$Percentage = '100%',
+
+        [ValidatePattern('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')]
+        [Parameter(ParameterSetName = 'Hex')]
+        [string]$BackgroundColor,
+
+        [ValidatePattern('^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$')]
+        [string]$FontColor = '#5f6982',
+
+        [ValidatePattern('^&#x[A-Fa-f0-9]{4,5}')]
+        [string]$IconHex
     )
     #$Script:HTMLSchema.Features.StatusButtonical = $true
-    if ($Icon -eq 'Dead') {
-        $IconType = 'performanceDead'
-    } elseif ($Icon -eq 'Bad') {
-        $IconType = 'performanceProblem'
-    } elseif ($Icon -eq 'Good') {
-        #$IconType = 'performance'
+    if ($IconHex) {
+        $IconType = $IconHex
```

Comment:
For icons you should allow a choice from Font Awesome like other commands are doing.